edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -139,7 +138,6 @@ def _verify_jwt_signature(token, jwt_issuer):
     try:
         _ = JWS().verify_compact(token, key_set)
     except Exception as exc:
-        logger.exception('Token verification failed.')
```

Comment:
@nasthagiri: This is the only message that I really need/want to remove.  The others I removed for consistency, assuming that the callers would handle logging.

Reply:
Reverted this change.